quipucordsctl/main.py

```diff
@@ -102,6 +104,11 @@ def main():
             print()  # new line for cleaner output before logger
             logger.error(_("Input closed unexpectedly."))
             sys.exit(1)
+        except (podman_errors.APIError, podman_errors.PodmanError):
+            # can occur if podman is not available or fails unexpectedly
+            print()  # new line for cleaner output before logger
+            logger.error(_("Communication with podman failed unexpectedly."))
+            sys.exit(1)
         except Exception as e:  # noqa: BLE001
             print()  # new line for cleaner output before logger
             logger.exception(e)
```

Comment:
Non blocker but... print by default writes to stdout while logger writes to stderr. I like the idea of using print to add line breaks, but maybe we should add `file=sys.stderr` (or something like this) to make sure we are writing everything to the same place.